src/main/Services/MainWindow.js

```
@@ -220,7 +222,22 @@ export class MainWindow {
       });
       this.window.webContents.on("did-finish-load", () => {
         this.window.addListener("resize", () => {
-          this.views.find((v) => v.isActive && !v.hidden)?.fit(!this.isToggled);
+          // Always use the latest sidebar width
+          const sidebarWidth = this.latestSidebarWidth || 250;
+          const bounds = this.window.getBounds();
+          const activeView = this.views.find((v) => v.isActive && !v.hidden);
+          if (activeView && activeView.view) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)